build/controllers/calcController.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
```

Comment:
Can "var" be replaced by "let" or "const"?
Are the functions '_createBinding, _setModuleDefault, _importStar, _importDefault' transpiled by typescript? Can the names be changed to something more descriptive?

Reply:
I'm not too sure about this one because this is what the compiler wrote when I used the tsc command. I don't want to touch the compiled script too much in case it explodes haha